.gitignore

```diff
@@ -21,3 +21,6 @@ pnpm-debug.log*
 
 # macOS-specific files
 .DS_Store
+
+# Local Netlify folder
+.netlify
```

Comment:
What lives in this folder? Nothing that needs to be shared, I’m guessing?

Reply:
Correct. This is for the case where a developer is using Netlify CLI so they can read our env vars. The CLI will generate local files for dev purposes, but we don't need them in prod.